src/utils/Utils.js

```diff
@@ -409,6 +409,31 @@ async function overrideParamsFromTestData(methodObj) {
   }
 }
 
+  // Aws api call to get the list of sdk version
```

Comment:
don't think this can be put in fca. aws plugin is not open source I believe

Reply:
moved this function to fca-parent utils file